### DISCUSSION TITLE
I cant log in

### DISCUSSION THREAD

**Discussion by noteveno** [vercel/community#5754] [Account Issue]
### Question

WHY I CANT LOG IN

**Comment by swarnava** [comment]
Hi, what is the error you are getting?

**Comment by noteveno** [comment]
when I try login (the first image)  ![image](https://github.com/vercel/community/assets/144538923/f5a78dc1-26e2-419b-bdcc-4cde558b4cfd)
when I try to signup I already have account (the second)  ![image](https://github.com/vercel/community/assets/144538923/417d864b-34ed-4a5a-b42e-b3626a26110a)
